docs/data-cookbooks/sns.md

```diff
@@ -68,15 +85,15 @@ sns:
         protocol: lambda
 ```
 
-Make sure that the receiver Lambda is configured in `lambdas.yml`.
+Make sure that the subscriber Lambda is configured in `lambdas.yml`. **Note that the Lambda name configured in `lambdas.yml`, `SnsS3Test`, needs to have its name postpended with `LambdaFunction` to have the ARN correctly found.**
```

Comment:
```suggestion
Make sure that the subscriber Lambda is configured in `lambdas.yml`. **Note that the Lambda name configured in `lambdas.yml`,`SnsS3Test`, needs to have its name postpended with `LambdaFunction` (similar to the example above) to have the ARN correctly found.**
```

Reply:
I think that something  to re-tie that statement to the above content might be warranted there.